helpdesk/overview/receiving_tickets.rst

```diff
@@ -0,0 +1,89 @@
+=======================
+Start Receiving Tickets
+=======================
+
+Offering a variety of channels from where your customers can contact you grants them flexibility,
+and the right to choose the best channel for themselves. And in order to make sure inquiries across
+all channels get addressed, it is important to have a solution where all interactions come together
+in one place.
+
+Channels options to submit tickets
+==================================
+
+Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`, and enable the following
+features as you want them to be available to your users.
+
+.. image:: media/channels_options.png
+   :align: center
+   :alt: View of a helpdesk teams setting page emphasizing the channels options in Odoo Helpdesk
+
+Email Alias
+-----------
+
+| Let your customers submit tickets by sending an email to your support email address. The subject
+  line of the email becomes the title of the ticket, and the email content is shown on the Chatter.
+| Select *Configure domain name* to be redirected to *Settings*, and from there, enable *External
+  Email Servers* to determine or change your *Alias Domain*.
+
+.. image:: media/channels_emailalias.png
+   :align: center
+   :height: 250
+   :alt: View of the settings page of a helpdesk team emphasizing the email alias feature
+         in Odoo Helpdesk
+
+.. note::
+   Using your own email server is required to send and receive emails in Odoo Community and
+   Enterprise. Online users benefit from a ready-to-use email server.
+
+Website Form
+------------
+
+| Allow your customers to submit a ticket by filling in a form through your website.
+| Once the feature is activated, get redirected to your website by clicking on *Go to Website*.
+
+.. image:: media/go_to_website.png
+   :align: center
+   :alt: View of the settings page of a helpdesk team emphasizing the Go to Website button in
+         Odoo Helpdesk
+
+From the website page, customize the form as you like, then choose to publish it by clicking
+on *Unpublished*.
```

Comment:
```suggestion
From the website page, customize the form as you like. Then, publish it by clicking
on *Unpublished*.
```

Reply:
I will keep it as one line (:

Reply:
I prefer his way. You can keep it as one line but choose to needs to be removed.

Reply:
ok